README.md

```diff
@@ -1,13 +1,39 @@
 # Camscripter Raspberry
```

Comment:
- [ ] camscripter.sh - smazat?
- [ ] systemd_file - co toj je za soubor? smazat?
- [ ] nemel by se systemd_register zavolat pri instalaci?
- [ ] v .sh scriptech ti chybi #/bin/bash
- [ ] html/shared slozka by nemela existovat, to je pozustatek z acap kde je to sdileny submodul
- [ ] v main.ts mas import z commondata a tam jsou vyjmenovany snad vsechny veci..neni lepsi dat import * ?

Reply:
html/shared jsme už myslím řešili a to UI je totalní spletenec lokalních a shared funkcí, rozplítat to se by zabralo zase bůhví kolik času a když to jen odstraním něco se bude zobrazovat blbě...

Reply:
nevím, ale mám pocit, že takhle ty importy v TS nefungujou... to by muselo být import * as neco a pak bych musel před každou tou věcí psat neco.blablabla...